examples/windows-audit/inspec.yml

```diff
@@ -0,0 +1,12 @@
+name: windows-audit
+title: Effortless Windows Audit
+maintainer: humans@habitat.sh
+copyright: Chef Software Inc.
+copyright_email: humans@habitat.sh
+license: Apache-2.0
+summary: Effortless Audit Windows Baseline
+version: 0.1.0
+
+depends:
+  - name: windows-baseline
+    url: https://github.com/dev-sec/windows-baseline/archive/1.2.0.tar.gz
```

Comment:
Any reason to pin to this specific version over master?

Reply:
Nope! I was just porting the existing version. Let's switch it

Reply:
Could we make the version tunable?

Reply:
If they tag and use the `git:` fetcher then yes.

Example: https://www.inspec.io/docs/reference/profiles/#git

Reply:
Though, profiles get vendored either via Effortless or when uploading to Automate. So using latest and testing via pipeline is preferred.

Reply:
I don't think I'd make that tunable for the sake of the example.

We'll do that when it comes to testing maybe?